dependencies.props

```diff
@@ -59,7 +59,7 @@
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>CoreFxVersion</ElementName>
-      <PackageId>Microsoft.Private.CoreFx.NETCoreApp</PackageId>
+      <PackageId>Microsoft.Private.CoreFx.UAP</PackageId>
```

Comment:
Can you follow my lead that I did in the release/2.0.0 branch and rename this? https://github.com/dotnet/core-setup/blob/release/2.0.0/dependencies.props#L21. We should probably also put those changes into master.